app/controllers/api/v1/credentials_controller.rb

```diff
@@ -3,7 +3,16 @@ class CredentialsController < ApiController
     respond_to :json
```

Comment:
No because we are using ```render json: credentials_serializer``` directly which makes the class level respond_to redundant. I'll remove it